docs/editor/integrated-terminal.md

```diff
@@ -164,6 +164,10 @@ The basics of the terminal have been covered in this document, read on to find o
 
 ```bat
 @echo off
+IF "%*" NEQ "" ( 
```

Comment:
Is this asking whether there are arguments with the command? Does CMD then run is under cmd.exe?

Reply:
Yes for both. Code calls the terminal with the arguments: "/d /c command_name command_args"

The /c flag indicates that cmd.exe should run and terminate

Reply:
Maybe would be better to check first if the arguments passed were in fact "/d /c" ? I will redo the PR.

Reply:
But doesn't cmder run some type of bash, not cmd? Is this meant to be a way to work around https://github.com/Microsoft/vscode/issues/36216?

Reply:
Yes, it felt strange to run CMD, so in the new commit I changed to just run the command, ignoring the _"/d /c"_ part of string. I don't think Cmder run a special bash or anything. 
I wasn't aware of the issue, but it seems to be caused by the same flags.